Framework/script/RepoCleaner/qcrepocleaner/rules/multiple_per_run.py

```diff
@@ -95,9 +103,19 @@ def process(ccdb: Ccdb, object_path: str, delay: int,  from_timestamp: int, to_t
         if policies_utils.in_grace_period(first_object, delay):
             logger.debug(f"     in grace period, skip this bucket")
             preservation_list.extend(run_versions)
-        elif not (from_timestamp < first_object.createdAt < to_timestamp):  # in the allowed period
+        elif not (from_timestamp < first_object.createdAt < to_timestamp):  # not in the allowed period
             logger.debug(f"     not in the allowed period, skip this bucket")
             preservation_list.extend(run_versions)
+        elif first_object.createdAtDt < datetime.now() - timedelta(minutes=mw_deletion_delay):
+            logger.debug(f"     after mw_deletion_delay period, delete this bucket")
+            for v in run_versions:
+                if "mw" in v.path: # this is because we really don't want to take the risk of batch deleting non moving windows
```

Comment:
```suggestion
                if "/mw/" in v.path: # this is because we really don't want to take the risk of batch deleting non moving windows
```
I would even go for "/mw/"`, otherwise you would include names such as `clusters_sumw2`.

Reply:
good point, thank you